From d8326da7a7721cbe51572478fa7d6559fa3dd740 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Mon, 28 Oct 2024 01:02:20 -0700
Subject: [PATCH 6/6] do not build examples by default

speed up the build by not building the
example applications by default with the
rocm sdk builder.

To enable the building of examples the
-DBUILD_EXAMPLES=1 must be defined

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bbf0670df..922c526aa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -474,7 +474,9 @@ if (NOT CK_BUILD_JIT_LIB)
                         LIBRARY_NAME composablekernel
                         PACKAGE_NAME examples
            )
-           add_subdirectory(example)
+           if(BUILD_EXAMPLES)
+                  add_subdirectory(example)
+           endif()
            if(BUILD_TESTING)
                   add_subdirectory(test)
            endif()
-- 
2.41.1

